Add osu!monthly #1 news post (#7680)
ppy/osu-wiki: news/2022-08-01-osumonthly-1.md

A best-of from this match was made on the [BTMC](https://osu.ppy.sh/users/3171691) [channel](https://twitch.tv/btmc), with exclusive interviews of the players, by [BTMC](https://osu.ppy.sh/users/3171691) (what a surprise, ahah) and Happystick, the casters of this amazing show! You can find it [here](https://www.youtube.com/watch?v=Y7LqBZL)!

Anyway, The Roundtable has now ended, we can't wait until the next edition!

<blockquote class="twitter-tweet"><p lang="en" dir="ltr">The Roundtable will return.<br>Thank you, everyone ❤️</p>&mdash; BTMC (@btmclive) <a href="https://twitter.com/btmclive/status/1546335465518682112?ref_src=twsrc%5Etfw">July 11, 2022</a></blockquote>

[*Find the news article here!*](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable)

Thanks again to [BTMC](https://osu.ppy.sh/users/3171691) and his staff for this insane event! 

### ![](/wiki/shared/mode/taiko.png) osu!taiko

#### EGTS 2022

> As if we could allow for osu! to take all the glory in its showcasing of top players and their performances, osu!taiko takes it upon itself to rival even that of [*The Roundtable*](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable) with their very own annual entry of **[Expert Global Taiko Showdown 2022](/wiki/en/Tournaments/GTS/EGTS_2022)**!

Reviewer: ```suggestion
As if we could allow for osu! to take all the glory in its showcasing of top players and their performances, members of the osu!taiko community take it upon themselves to rival even that of [*The Roundtable*](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable) with their very own annual entry of **[Expert Global Taiko Showdown 2022](/wiki/en/Tournaments/GTS/EGTS_2022)**!
```
Reads a bit better to me, as these events are made possible by the community and not through the game mode. Using the pronouns consistently was my main concern here (see `itself`, `themselves` and `their`).

<iframe width="100%" height="315" src="https://youtu.be/embed/1RuyJ3y_ot4" frameborder="0" allowfullscreen></iframe>

#### Millions of Long Notes

Wow. Just completely shocked at this performance by [[GS]Myuka](https://osu.ppy.sh/users/10072733) on [Camellia’s - MI11I0N PP](https://osu.ppy.sh/beatmapsets/1388481#mania/2867552). 

If you thought *My Man* had LNs, wait until you see a map of over 7 minutes of non-stop LN gameplay, and with not just single-note rainbows and glissandos, but an interconnected web of the two. It is an impeccable display of skill, accuracy, speed, and just what osu!monthly is looking for in a showcase!

<iframe width="100%" height="315" src="https://youtu.be/embed/nc_ejGtZKUM" frameborder="0" allowfullscreen></iframe>

### ![](/wiki/shared/mode/catch.png) osu!catch

#### Galaxy Brain

Somebody scream! Even with being the number one osu!catch player in the world, a play such as [YesMyDarkness’](https://osu.ppy.sh/users/4158549/fruits) is no small feat. 

> Coming in with a HD HR 4xmiss on [Kurokotei’s Galaxy Collapse](https://osu.ppy.sh/beatmapsets/521900#fruits/1108507) one of the most well known artists and songs in osu! surely does do them some justice. Riddled with intense and **dash**ingly hard jumps it is a play to contend with even the hardest of maps, let alone with hidden and hardrock! 

Reviewer 2: ``` Suggestion
Coming in with an HD HR 4xmiss on [Kurokotei’s Galaxy Collapse](https://osu.ppy.sh/beatmapsets/521900#fruits/1108507) one of the most well-known artists and songs in osu! surely does do them some justice. Riddled with intense and **dash**ingly hard jumps it is a play to contend with even the hardest of maps, let alone with hidden and hardrock!
```

<iframe width="100%" height="315" src="https://www.youtube.com/embed/byyHOAopo1A" frameborder="0" allowfullscreen></iframe>

---

## New Features on osu!(lazer)!

> The new client has never so **close** from releasing (there's still a lot of work on it!), and has **new mods**, more original than never! 

Reviewer: ```suggestion
The new client has never been so **close** to release (there's still a lot of work on it!), and has **new mods**, more original than never!
```

> For the first time ever, lazer patch notes have been posted on the official osu! [YouTube channel](https://www.youtube.com/c/osugame), compiled by [pishifat](https://osu.ppy.sh/users/3178418) with further insight provided by the man himself, [peppy](https://osu.ppy.sh/users/2)!

Reviewer: ```suggestion

For the first time ever, lazer patch notes have been posted on the official osu! [YouTube channel](https://www.youtube.com/c/osugame), compiled by [pishifat](https://osu.ppy.sh/users/3178418) with further insight provided by the man himself, [peppy](https://osu.ppy.sh/users/2)!
```
Should be a separate paragraph in my opinion (markdown will otherwise show it as one).

One thing you might've missed is the new fancy Repel mod added to lazer, which is pretty chaotic but fun (finally a worthy opponent for [badeu](https://osu.ppy.sh/users/1473890)?)

<div align="center">
    <video width="100%" controls>
        <source src="https://user-images.githubusercontent.com/47010459/172435116-6b09a48c-6fd6-4abb-9e5f-e96ab3565184.mp4" type="video/mp4" preload="none">
    </video>
</div>

Another feature that's maybe not the newest, but some of you might have missed, or just haven't gotten around to trying yet is the **skin layout editor**! First implemented in May of 2021, this is still very much a work-in-progress feature, however currently you are able to edit the placement of all skin elements that appear during gameplay! Check out the showcase video from the original patch notes below:

<div align="center">
    <video width="100%" controls>
        <source src="https://user-images.githubusercontent.com/191335/118155608-5c5fce80-b453-11eb-88f4-e552b10ed5f1.mp4" type="video/mp4" preload="none">
    </video>
</div>

## Month's Community Quiz

> A monthly question or puzzle for you!

Reviewer 2: ```Suggestion
For those who think they have all the answers... here's a riddle from us to you!
```

> Once upon a time, in the ye' old osu!, someone asked for advice from the ancestors among osu!, the post reads as follows: 

> "Is it possible to repeat songs in osu!? I have an in-game video of KH2 with Roxas battling the Twilight Thorn (people who have played Kingdom Hearts 2 will be drooling) and the video is still halfway when the song ends! Or can someone at least tell me about a free, easy to use software which can solve my problem by repeating the song?"

> This ancient tome is hidden behind a code, the code you seek is everywhere but no where at the same time, always in the back of your mind but only breaching the surface at the moment you see it, now... go forth, unveil the secret code, and paste the number in the link, and gain access to this ancient tome!

> > The first person to answer it correctly will receive a month of osu!supporter, leave your answer in the comment!

Reviewer: ```suggestion
The first person to answer it correctly will receive a month of osu!supporter, leave your answer in the comment!
```
I recommend to remove the block quotation for this one. It feels less as a part of the quiz itself and more as a piece for the user prompt.

## End Word

> We wanted to thank the osu!team and the contributors to [osu!lazer](https://github.com/ppy/osu), [osu!web](https://github.com/ppy/osu-Web) and [osu!wiki](https://github.com/ppy/osu-wiki) for their hard work on the repositories and the trust thay gave us. Huge thanks again to [Nelly](https://osu.ppy.sh/users/4741164) for the osu!catch news bulletin, [fajar13k](https://osu.ppy.sh/users/7100002) for his contribution to the EGTS part, and to [The-Last-Cookie](https://github.com/The-Last-Cookie) for his amazing review (a lot of things to correct, right?)!

Reviewer: ```suggestion
We wanted to thank the osu!team and the contributors to [osu!lazer](https://github.com/ppy/osu), [osu!web](https://github.com/ppy/osu-Web) and [osu!wiki](https://github.com/ppy/osu-wiki) for their hard work on the repositories and the trust thay gave us. Huge thanks again to [Nelly](https://osu.ppy.sh/users/4741164) for the osu!catch news bulletin, [fajar13k](https://osu.ppy.sh/users/7100002) for his contribution to the EGTS part, and to [The_Last_Cookie](https://osu.ppy.sh/users/11587828) for his amazing review (a lot of things to correct, right?)!
```
I would like the article to link to my osu! profile. Thank you very much for the mention! ^^

> If you're interested on seeing the creation processus, the backstages of osu!, please join the [osu!dev server](https://discord.gg/ppy)! (Articles from external contributors are welcome)

Reviewer: ```suggestion
If you're interested in seeing the creation processus, the backstages of osu!, please join the [osu!dev server](https://discord.gg/ppy)! (Articles from external contributors are welcome)
```
`interested on` should be [avoided](https://grammarhow.com/interested-in-or-on/).

> If you're interested on seeing the creation processus, the backstages of osu!, please join the [osu!dev server](https://discord.gg/ppy)! (Articles from external contributors are welcome)

Reviewer 2: ``` Suggestion
If you're interested in seeing the creation process, and the backstages of osu!, please join the [osu!dev server](https://discord.gg/ppy)! (Articles from external contributors are welcome)
```

Reviewer: Just as an aside note, because it interested me: [processus](https://dictionary.cambridge.org/us/dictionary/english/processus) is a real English word. However, I agree with your suggestion, as it is more readable and I believe that `processus` is not a common word.

While waiting for the next article, we want to thank in advance all of the community members who'll send us messages about subjects you want covered. If you have any feedback or thoughts about the osu!monthly, pour it all in the forum [discussion thread](https://osu.ppy.sh/community/forums/topics/1614314)!

> It's a real honor for us to write for you, the community. Thanks for reading, and see you soon for the next article, in September! ❤️

Reviewer 2: ``` Suggestion
It's a real honour for us to write for you, the community. Thanks for reading, and see you soon for the next article, in September! ❤️
```

---
layout: post
title: osu!monthly #1
date: 2022-07-28
author: Soetch, Blushing, wowcake, 0x84f
---

This is the new community's news post : osu!monthly (a new one, again)!

![](/wiki/shared/news/2022-08-01-osumonthly-1/banner-july.png)

> ## Word of the Redaction : 

Reviewer: ```suggestion
## Word of the editors
```

Author: Correcting it too.

> It's been years since **osu!weekly's** release has been paused, 6 years since the community didn't have a regular post, or almost not. Today, we're coming in with **__osu!monthly__**, a bigger format. 

Reviewer: ```suggestion

It's been years since **osu!weekly's** release has been paused, 6 years since the community didn't have a regular post. Today, we're coming in with **__osu!monthly__**, a bigger format. 
```

Author: Yep, correcting it!

A best-of from this match was made on the BTMC channel, with exclusive interviews of the players, by BTMC (what a surprise, ahah) and Happystick, the casters of this amazing show! You can find it [here](https://www.youtube.com/watch?v=Y7LqBZL)!

Anyway, The Roundtable has now ended, and we wait impatiently the next edition, next year maybe ?

[➥ *Find the forum article here!*](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable) 

Thanks again to [BTMC](https://osu.ppy.sh/users/3171691) and his staff for this insane event! 

### ![](/wiki/shared/mode/mania.png) osu!mania
#### In August, the osu!mania top players will brawl in the *osu!mania World Cup 2022*!

As it is starting the **6th of August**, we can't actually talk about it more, but we are excited to say that the next osu!monthly is going to have a **big article** about it, and maybe an **interview of the winner**!

Please stay tuned about the next article, and we're moving on the next section!

> [➥ *Find the forum article here!*](https://osu.ppy.sh/home/news/2022-07-07-osu-mania-4k-world-cup-2022-registrations-now-open) 

Reviewer: ```suggestion
[*Find the forum article here!*](https://osu.ppy.sh/home/news/2022-07-07-osu-mania-4k-world-cup-2022-registrations-now-open) 
```

> 

Reviewer: ```suggestion
```

### ![](/wiki/shared/mode/catch.png) osu!catch

> #### At the beginning of July, the recent eleventh iteration of osu!catch World Cup has been concluded.

Reviewer: ```suggestion

*At the beginning of July, the recent eleventh iteration of osu!catch World Cup has been concluded.*
```
Seems a bit long for a title to me. And it's the only title for this section anyway.

With the recent change of swapping from Group Stages to Qualifiers format, most of the players and viewers didn't expect to see China on a very top despite failing their previous run placing 8th-7th place. Their expectations worth it in the end as they instantly gained their glory reaching winners finals bracket without losing a single match until meeting with South Korea and untimately to United States. Even with a long year absense, China showed that even being absent for an entire year they can still make a good fight with top teams.

There's still plenty of tournaments running though even after the conclusion of CWC! The digit tournaments, 4 Digit Catch World Cup 2022 and 3 Digit Catch World Cup 2022, are up and running and a second iteration of Catch French Dual Cup 2 currently in the registered phase. There's nothing to talk about these tournaments yet but maybe with the new osu!monthly release we can fully recap what was going on with conclusions and stuff!

### ![](/wiki/shared/mode/taiko.png) osu!taiko

>  #### As if we could allow for osu! to take all the glory in its showcasing of top players and their performances, Taiko takes it upon themself to rival even that of *The Roundtable* with their very own annual entry of **[Expert Global Taiko Showdown 2022](https://osu.ppy.sh/wiki/en/Tournaments/GTS/EGTS_2022)**! 

Reviewer: ```suggestion

As if we could allow for osu! to take all the glory in its showcasing of top players and their performances, osu!taiko takes it upon itself to rival even that of *The Roundtable* with their very own annual entry of **[Expert Global Taiko Showdown 2022](https://osu.ppy.sh/wiki/en/Tournaments/GTS/EGTS_2022)**!
```
I'm very sure that you mean the game mode. In this case, I would suggest to write that the taiko community is also engaged in tournaments and is making efforts in this regard.

Author: Yep!

As the name implies, the showdown is to take place between those whose dreams consist of taiko drumming, kiai slider velocity shenanigans, dons, katsus, and if we even dare-say, taikonator 3000! This is a tournament that showcases those whose passion for Taiko is not just exemplary in their gameplay but in every other way. 

---

## Top scores of the Month!

> ### ![](/wiki/shared/mode/osu.png) osu! 

Reviewer: ```suggestion

### ![](/wiki/shared/mode/osu.png) osu!
```

> #### As if the impossible could not be done only once, but twice!? 

Reviewer: ```suggestion

#### As if the impossible could not be done only once, but twice!?
```

> [[Karcher]](https://osu.ppy.sh/users/11443437) is an absolute madman! Achieving an insane Full Combo on [*Yomi yori Kikoyu, Koukoku no Tou to Honoo no Shoujo*](https://osu.ppy.sh/beatmapsets/461744#fruits/1031991) to ever hit the ranked section. For just over 8 minutes of gameplay at a staggering 220 BPM it is no small feat, and thus, should be showed off for osu!’s monthly pick! 

Reviewer: ```suggestion

[\[Karcher\]](https://osu.ppy.sh/users/11443437) is an absolute madman! Achieving an insane Full Combo on [*Yomi yori Kikoyu, Koukoku no Tou to Honoo no Shoujo*](https://osu.ppy.sh/beatmapsets/461744#fruits/1031991) to ever hit the ranked section. For just over 8 minutes of gameplay at a staggering 220 BPM it is no small feat, and thus, should be showed off for osu!’s monthly pick!
```
Not sure about formatting here, but in wiki articles square brackets (`[` and `]`) need to be escaped, so they aren't interpreted by markdown.

Author: Wasn't sure about it, thanks for confirming!

---
layout: post
title: osu!monthly #1
date: 2022-07-28

> author: Soetch, Blushing, wowcake, 0x84f

Author: Waiting for a confirmation.

---

This is the new community's news post : osu!monthly (a new one, again)!

![](/wiki/shared/news/2022-08-01-osumonthly-1/banner-july.png)

## Word of the editors

It's been years since **osu!weekly's** release has been paused, 6 years since the community didn't have a regular post. Today, we're coming in with **__osu!monthly__**, a bigger format.

We think we did a good job gathering information for **July**, but we started the 07/17, so we had only two week to write, review and commit this article. 

It'll never be the same as osu!weekly, but we're working hard to propose a frequent format, please be kind with us, new writers! We're highly receptive to it, so feedback is highly appreciated!

➥ *[Soetch](https://osu.ppy.sh/users/15970028), [Blushing](https://osu.ppy.sh/users/5927823), [wowcake](https://osu.ppy.sh/users/16121851) & [0x84f](https://osu.ppy.sh/users/7944724)*

> ---

Author: Agreed.

### Navigation

> - [What's up on osu! ?](#whats-up-on-osu)

Author: Fixed.

- [Top Scores of the Month](#top-scores-of-the-month)
- [New features on osu!lazer (& bonus article!)](#new-features-on-osulazer)
- [Summary of this article](#summary-of-this-article)
- [End Word](#end-word)

---

> ## What's up on osu! ? 

Author: Fixed.

> ## What's up on osu! ? 

Reviewer: I know this was used a few times in the article, but I'm not sure about placing question marks with a space in front of them. On the wiki, there are plenty example of `osu!?`.

- *Latest updates, events… and even more!*

### ![](/wiki/shared/mode/osu.png) osu! 

> #### If you ask us what is the big event of July, it'd be the **Roundtable**!

Reviewer: ```suggestion

#### If you ask us what is the big event of July, it'd be the Roundtable!
```
Headings may not have formatting at least in wiki articles.

<br>

![](/wiki/shared/news/2022-08-01-osumonthly-1/roundtable/RTable_title.png)
![](/wiki/shared/news/2022-08-01-osumonthly-1/roundtable/RTable_players.jpg)

> With the players showcased above, we have [BTMC](https://osu.ppy.sh/users/3171691) and [Happystick](https://osu.ppy.sh/users/256802) commentating, a genius roster who make sense for this first **IRL osu! invitational**! 

Reviewer: ```suggestion
With the players showcased above, we have [BTMC](https://osu.ppy.sh/users/3171691) and [Happystick](https://osu.ppy.sh/users/256802) commentating, genius roster who make sense for this first **IRL osu! invitational**!
```

> As if it wasn't enough, [mrekk](https://osu.ppy.sh/users/7562902) is there too! If you weren't able to follow the event, no worries! The official Roundtable's YouTube channel has the rediffusions [here](https://www.youtube.com/channel/UC6fSQd4oKBPxkBR93GucCUg/featured)!

Author: Fixed.

With that many great players, such as [mrekk](https://osu.ppy.sh/users/7562902), [Utami](https://osu.ppy.sh/users/7512553), [RyuK](https://osu.ppy.sh/users/6304246) and many others, this tournament was incredible! The competition's spirit was mixed with the entertaining cast and gimmicks of [BTMC](https://osu.ppy.sh/users/3171691) and the result was epic!

In terms of epic, we were served with the finals : [mrekk](https://osu.ppy.sh/users/7562902) vs [Utami](https://osu.ppy.sh/users/7512553)! Despite mrekk's defeat against Utami, his performances were impressive, along with Utami's. His first rank on the leaderboard is indeed his throne.

A best-of from this match was made on the BTMC channel, with exclusive interviews of the players, by BTMC (what a surprise, ahah) and Happystick, the casters of this amazing show! You can find it [here](https://www.youtube.com/watch?v=Y7LqBZL)!

> Anyway, The Roundtable has now ended, and we wait impatiently the next edition, next year maybe ?

Reviewer: ```suggestion
Anyway, The Roundtable has now ended, and hopefully this towering experience returns next year!
```

> [➥ *Find the forum article here!*](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable) 

Reviewer: ```suggestion
[*Find the forum article here!*](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable)
```

Thanks again to [BTMC](https://osu.ppy.sh/users/3171691) and his staff for this insane event! 

### ![](/wiki/shared/mode/mania.png) osu!mania

> #### In August, the osu!mania top players will brawl in the *osu!mania World Cup 2022*!

Reviewer: ```suggestion

#### In August, the osu!mania top players will brawl in the osu!mania World Cup 2022!
```

<iframe width="100%" height="315" src="<link>https://youtu.be/embed/hKwIKC-DvcU" frameborder="0" allowfullscreen></iframe>

### ![](/wiki/shared/mode/mania.png) osu!mania

#### The mania k-pop community is going to be absolutely ecstatic at this next showcase. 
With [Onta_Bekasi](https://osu.ppy.sh/users/8677684) not just clearing, but FCing the deceptively difficult yet soothing myriad of LNs and intricate patterns that [My Man](https://osu.ppy.sh/beatmapsets/514681#mania/1093212) so elegantly puts on display, it is of true bewilderment that [Onta_Bekasi](https://osu.ppy.sh/users/8677684) only suffers a 00.02 accuracy drop across a 3 minute map and over 5000 notes. Truly a wonderful display of skill, enjoyment of the game, and a play that is more than deserving of being showcased.

<iframe width="100%" height="315" src="<mania 7k>https://youtu.be/embed/1RuyJ3y_ot4" frameborder="0" allowfullscreen></iframe>

#### Wow. Just completely shocked at this performance by [[GS]Myuka](https://osu.ppy.sh/users/10072733) on [Camellia’s - MI11I0N PP](https://osu.ppy.sh/beatmapsets/1388481#mania/2867552). 

> If you thought *My Man* had LNs, wait til you watch just over 7 minutes of almost non-stop LN action along with being bridled by intense single-note action and 

Author: Fixed.

BPM changes throughout the map. 

<iframe width="100%" height="315" src="<mania 4k>https://youtu.be/embed/nc_ejGtZKUM" frameborder="0" allowfullscreen></iframe>

### ![](/wiki/shared/mode/catch.png) osu!catch
#### Somebody scream! Even with being the number one osu!catch player in the world, a play such as [YesMyDarkness’](https://osu.ppy.sh/users/4158549/fruits) is no small feat. 
Coming in with a HD HR 4xmiss on [Kurokotei’s Galaxy Collapse](https://osu.ppy.sh/beatmapsets/521900#fruits/1108507) one of the most well known artists and songs in osu! surely does do them some justice. Riddled with intense and **dash**ingly hard jumps it is a play to contend with even the hardest of maps, let alone with hidden and hardrock! 

<iframe width="100%" height="315" src="https://www.youtube.com/embed/byyHOAopo1A" frameborder="0" allowfullscreen></iframe>

---
## New Features on osu!lazer!
The new client has never been that **close** from releasing (there's still a lot of work on it!), and has **new mods**, more original than never! 

> Here's the [patch note](https://osu.ppy.sh/home/changelog), and the showcase of **the latest mod**! 

Author: Fixed.

To start with this, we want to showcase a pretty chaotic but fancy Repel mod (finally a worthy opponent for badeu?)! 

<div align="center">
    <video width="100%" controls>
        <source src="https://user-images.githubusercontent.com/47010459/172435116-6b09a48c-6fd6-4abb-9e5f-e96ab3565184.mp4" type="video/mp4" preload="none">
    </video>
</div>

---
That said, you're maybe thinking that this Lazer section isn't long enough. Right ? Here's a section about Lazer's UI design, wrote by [0x84f](https://osu.ppy.sh/users/7944724).

## Summary of this Article
### ![](/wiki/shared/mode/osu.png) osu!
- The Roundtable has ended on insane matches, especially the Utami vs mrekk one! 
  
  [➥ Find the article here!](#if-you-ask-us-what-is-the-big-event-of-july-itd-be-the-roundtable)

- For the month's top score, we got [[Karcher]](https://osu.ppy.sh/users/11443437) achieving an insane Full Combo on [*Yomi yori Kikoyu, Koukoku no Tou to Honoo no Shoujo*](https://osu.ppy.sh/beatmapsets/461744#fruits/1031991)!
  
  [➥ Find the article here!](#as-if-the-impossible-could-not-be-done-only-once-but-twice)
  

### ![](/wiki/shared/mode/mania.png) osu!mania

> - The osu!mania World Cup 2022 is the road to the greatness for the Mania top players! They will brawl again in this great arena! Who's gonna take the crown ?

Author: Fixed.

  
  [➥ Find the article here!](#in-august-the-osumania-top-players-will-brawl-in-the-osumania-world-cup-2022)

- The scores now!
  Let's start with a 7K score : Onta_Bekasi only lost 0,02 accuracy on the surprisingly difficult *"My Man"*! A flawless performance!

  [➥ Find the article here!](#the-mania-k-pop-community-is-going-to-be-absolutely-ecstatic-at-this-next-showcase)

- Now, an amazing 4K score! If you thought *My Man* had LNs, wait 'till you watch just over 7 minutes of almost non-stop LN action along with being bridled by intense single-note action and 
BPM changes throughout the map. [GS] Myuka has done a rampage on the Camellia’s - MI11I0N PP]!

  [➥ Find the article here!](#wow-just-completely-shocked-at-this-performance-by-gsmyuka-on-camellias---mi11i0n-pp)

### ![](/wiki/shared/mode/catch.png) osu!catch

> - The Catch World Cup 2022 has ended on pretty amazing performances ! Is South Korea dominating two modes at the same time ?

Reviewer 3: - The Catch World Cup 2022 has ended on pretty amazing performances ! Is South Korea dominating two modes at the same time?

Author: Fixed.

  
  [➥ Find the article here!](#at-the-beginning-of-july-the-recent-eleventh-iteration-of-osucatch-world-cup-has-been-concluded)

> - The score of the month, according to us, is the [YesMyDarkness’](https://osu.ppy.sh/users/4158549/fruits) HD HR 4xmiss on [Kurokotei’s Galaxy Collapse](https://osu.ppy.sh/beatmapsets/521900#fruits/1108507)! Everybody screeeeeeeeeeeeeeeeeam!

Author: Fixed (thought it was though).

  
  [➥ Find the article here!](#everybody-scream-even-with-being-the-number-one-osucatch-player-in-the-world-a-play-such-as-yesmydarkness-is-no-small-feat)

### ![](/wiki/shared/mode/taiko.png) osu!taiko 
- The Expert Global Taiko Showdown 2022 is hitting on the stage! This is the way Taiko players can overtake [The Roundtable](#if-you-ask-us-what-is-the-big-event-of-july-itd-be-the-roundtable) in terms of epic shows!
  
  [➥ Find the article here!](#as-if-we-could-allow-for-osu-to-take-all-the-glory-in-its-showcasing-of-top-players-and-their-performances-taiko-takes-it-upon-themself-to-rival-even-that-of-the-roundtable-with-their-very-own-annual-entry-of-expert-global-taiko-showdown-2022)

- With all greatness comes great scores! This month, we have [shinchikuhome’s](https://osu.ppy.sh/users/3174184) DT play on [Apollo](https://osu.ppy.sh/beatmapsets/1739877#taiko/3558252)! **411 BPM!** That's for you, speed enjoyers!
  
  [➥ Find the article here!](#whew-if-you-thought-the-pick-for-osu-was-crazy-wait-till-you-watch-shinchikuhomes-dt-play-on-apollo)

## Month's Community Quiz

> A monthly question or puzzle for you !

Author: Fixed.

>This ancient tome is hidden behind a code, the code you seek is everywhere but no where at the same time, always in the back of your mind but only breaching the surface at the moment you see it, now… go forth, unveil the secret code, and paste the number in the link, and gain access to this ancient tome!

>The first person to answer it correctly will receive a month of osu!supporter, leave your answer in the comment!

## End Word
We wanted to thank the osu!team and the contributors to [osu!lazer](https://github.com/ppy/osu), [osu!web](https://github.com/ppy/osu-Web) and [osu!wiki](https://github.com/ppy/osu-wiki) for their hard work on the repositories and the trust thay gave us. Huge thanks again to [Nelly](https://osu.ppy.sh/users/4741164) for the osu!catch news bulletin!

If you're interested on seeing the creation processus, the backstages of osu!, please join the [osu!dev server](https://discord.gg/ppy)! (Articles from external contributors are welcome)

> While waiting for the next article, we thank in advance all of the ones who'll send us messages about subjects you want us to cover. If you have any feedback or thoughts about the osu!monthly, pour it all in the forum [discussion thread]()!

Author: Fixed.

>The first person to answer it correctly will receive a month of osu!supporter, leave your answer in the comment!

## End Word
We wanted to thank the osu!team and the contributors to [osu!lazer](https://github.com/ppy/osu), [osu!web](https://github.com/ppy/osu-Web) and [osu!wiki](https://github.com/ppy/osu-wiki) for their hard work on the repositories and the trust thay gave us. Huge thanks again to [Nelly](https://osu.ppy.sh/users/4741164) for the osu!catch news bulletin!

If you're interested on seeing the creation processus, the backstages of osu!, please join the [osu!dev server](https://discord.gg/ppy)! (Articles from external contributors are welcome)

While waiting for the next article, we want to thank in advance all of the community members who'll send us messages about subjects you want covered. If you have any feedback or thoughts about the osu!monthly, pour it all in the forum [discussion thread]()!

It's a real honor for us to write for you, the community. Thanks for reading, and see you soon for the next article, in September! ❤️

---

> ➥ *[Soetch](https://osu.ppy.sh/users/15970028), [Blushing](https://osu.ppy.sh/users/5927823), [wowcake](https://osu.ppy.sh/users/16121851), [0x84f](https://osu.ppy.sh/users/7944724)  & [Hubz](https://osu.ppy.sh/users/10379965) for this AMAZING banner!*

Reviewer: ```suggestion
—[Soetch](https://osu.ppy.sh/users/15970028), [Blushing](https://osu.ppy.sh/users/5927823), [wowcake](https://osu.ppy.sh/users/16121851), [0x84f](https://osu.ppy.sh/users/7944724) & [Hubz](https://osu.ppy.sh/users/10379965) for this AMAZING banner!
```

Anyway, The Roundtable has now ended, and hopefully this towering experience returns next year!

[*Find the forum article here!*](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable)

Thanks again to [BTMC](https://osu.ppy.sh/users/3171691) and his staff for this insane event! 

### ![](/wiki/shared/mode/taiko.png) osu!taiko

As if we could allow for osu! to take all the glory in its showcasing of top players and their performances, osu!taiko takes it upon itself to rival even that of *The Roundtable* with their very own annual entry of **[Expert Global Taiko Showdown 2022](https://osu.ppy.sh/wiki/en/Tournaments/GTS/EGTS_2022)**!

As the name implies, the showdown is to take place between those whose dreams consist of taiko drumming, kiai slider velocity shenanigans, dons, katsus, and if we even dare-say, taikonator 3000! This is a tournament that showcases those whose passion for Taiko is not just exemplary in their gameplay but in every other way. 

### ![](/wiki/shared/mode/catch.png) osu!catch

*At the beginning of July, the recent eleventh iteration of osu!catch World Cup has been concluded.*

> For the fourth time in a row and a total of eight wins, South Korea are the osu!catch champions. The roster consisting of JISAN, Rells, YesMyDarknesss, x_angelkawaii_x, DreStar and Byeol. With second and third places placed by United States (Secre, Lexii, Dahcreeper, Zak, Colin, Elux) and China (Dusk, Callionet, Coevir, Crystal, KagamineRin02, Kitakami).

Reviewer 4: For the fourth time in a row and a total of eight wins, South Korea are the osu!catch champions. The roster consists of [JISAN](https://osu.ppy.sh/users/3657951), [Rells](https://osu.ppy.sh/users/7937519), [YesMyDarknesss](https://osu.ppy.sh/users/4158549), [x_angelkawaii_x](https://osu.ppy.sh/users/566276), [DreStar](https://osu.ppy.sh/users/1808057) and [Byeol](https://osu.ppy.sh/users/2754651). With second and third places placed by United States ([Secre](https://osu.ppy.sh/users/2306637), [Lexii](https://osu.ppy.sh/users/7226149), [Dahcreeper](https://osu.ppy.sh/users/6926006), [Zak](https://osu.ppy.sh/users/1375955), [Colin](https://osu.ppy.sh/users/5502521), [Elux](https://osu.ppy.sh/users/12004983)) and China ([Dusk](https://osu.ppy.sh/users/533210), [Callionet](https://osu.ppy.sh/users/3072921), [Coevir](https://osu.ppy.sh/users/3171965), [Crystal](https://osu.ppy.sh/users/1646397), [KagamineRin02](https://osu.ppy.sh/users/518503), [Kitakami](https://osu.ppy.sh/users/6347308)).

---
layout: post
title: osu!monthly #1
date: 2022-07-28
author: Soetch, Blushing, wowcake, 0x84f
---

![](/wiki/shared/news/2022-08-01-osumonthly-1/banner-july.png)

# Introducing osu!monthly

It has been a long time since the last osu!weekly was released and luckily, we have something exciting to show you: osu!monthly! The new community news post format.

> 6 years passed in which the community didn't have a regular post. Today, we're coming in with ***osu!monthly***, a bigger format.

Author: agreed.

## Word of the editors :

> We think we did a good job gathering information for **July**, but we started on the 17 July, so we had only two weeks to write, review and commit this article.

Author: nice one, blushingW

With that many great players, such as [mrekk](https://osu.ppy.sh/users/7562902), [Utami](https://osu.ppy.sh/users/7512553), [RyuK](https://osu.ppy.sh/users/6304246) and many others, this tournament was incredible! The competition's spirit was mixed with the entertaining cast and gimmicks of [BTMC](https://osu.ppy.sh/users/3171691) and the result was epic!

In terms of epic, we were served with the finals: [mrekk](https://osu.ppy.sh/users/7562902) vs [Utami](https://osu.ppy.sh/users/7512553)! Despite mrekk's defeat against [Utami]([Utami](https://osu.ppy.sh/users/7512553)), his performances were impressive, along with [Utami](https://osu.ppy.sh/users/7512553)'s. His first rank on the leaderboard is indeed his throne.

A best-of from this match was made on the [BTMC](https://osu.ppy.sh/users/3171691) [channel](https://twitch.tv/btmc), with exclusive interviews of the players, by [BTMC](https://osu.ppy.sh/users/3171691) (what a surprise, ahah) and Happystick, the casters of this amazing show! You can find it [here](https://www.youtube.com/watch?v=Y7LqBZL)!

> Anyway, The Roundtable has now ended, and hopefully this towering experience returns next year!

Reviewer 2: ``` Suggestion
Anyway, The Roundtable has now ended, and hopefully, this towering experience returns next year!
```

[*Find the forum article here!*](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable)

Thanks again to [BTMC](https://osu.ppy.sh/users/3171691) and his staff for this insane event! 

### ![](/wiki/shared/mode/taiko.png) osu!taiko

As if we could allow for osu! to take all the glory in its showcasing of top players and their performances, osu!taiko takes it upon itself to rival even that of [*The Roundtable*](https://osu.ppy.sh/home/news/2022-07-08-the-roundtable) with their very own annual entry of **[Expert Global Taiko Showdown 2022](https://osu.ppy.sh/wiki/en/Tournaments/GTS/EGTS_2022)**!

As the name implies, the showdown is to take place between those whose dreams consist of taiko drumming, kiai slider velocity shenanigans, dons, katsus, and if we even dare-say, taikonator 3000! This is a tournament that showcases those whose passion for Taiko is not just exemplary in their gameplay but in every other way. 

### ![](/wiki/shared/mode/catch.png) osu!catch

*At the beginning of July, the recent eleventh iteration of the [osu!catch World Cup 2022](https://osu.ppy.sh/wiki/en/Tournaments/CWC/2022) has been concluded.*

![](/wiki/shared/news/2022-08-01-osumonthly-1/podium.png)

> For the fourth time in a row and a total of eight wins, South Korea are the osu!catch champions. The roster consisting of [JISAN](https://osu.ppy.sh/users/3657951), [Rells](https://osu.ppy.sh/users/7937519), [YesMyDarknesss](https://osu.ppy.sh/users/4158549), [x_angelkawaii_x](https://osu.ppy.sh/users/566276), [DreStar](https://osu.ppy.sh/users/1808057) and [Byeol](https://osu.ppy.sh/users/2754651). With second and third places placed by United States ([Secre](https://osu.ppy.sh/users/2306637), [Lexii](https://osu.ppy.sh/users/7226149), [Dahcreeper](https://osu.ppy.sh/users/6926006), [Zak](https://osu.ppy.sh/users/1375955), [Colin](https://osu.ppy.sh/users/5502521), [Elux](https://osu.ppy.sh/users/12004983)) and China ([Dusk](https://osu.ppy.sh/users/533210), [Callionet](https://osu.ppy.sh/users/3072921), [Coevir](https://osu.ppy.sh/users/3171965), [Crystal](https://osu.ppy.sh/users/1646397), [KagamineRin02](https://osu.ppy.sh/users/518503), [Kitakami](https://osu.ppy.sh/users/6347308)).

Reviewer 2: ``` Suggestion
For the fourth time in a row and a total of eight wins, South Korea are the osu!catch champions. The roster consisted of [JISAN](https://osu.ppy.sh/users/3657951), [Rells](https://osu.ppy.sh/users/7937519), [YesMyDarknesss](https://osu.ppy.sh/users/4158549), [x_angelkawaii_x](https://osu.ppy.sh/users/566276), [DreStar](https://osu.ppy.sh/users/1808057), and [Byeol](https://osu.ppy.sh/users/2754651). With second and third places won by the United States ([Secre](https://osu.ppy.sh/users/2306637), [Lexii](https://osu.ppy.sh/users/7226149), [Dahcreeper](https://osu.ppy.sh/users/6926006), [Zak](https://osu.ppy.sh/users/1375955), [Colin](https://osu.ppy.sh/users/5502521), [Elux](https://osu.ppy.sh/users/12004983)) and China ([Dusk](https://osu.ppy.sh/users/533210), [Callionet](https://osu.ppy.sh/users/3072921), [Coevir](https://osu.ppy.sh/users/3171965), [Crystal](https://osu.ppy.sh/users/1646397), [KagamineRin02](https://osu.ppy.sh/users/518503), [Kitakami](https://osu.ppy.sh/users/6347308)), respectively.
```

> It's not a secret that players from South Korea are so good that they could swap each other and win the entire tournament no matter what's ahead. With no [qwhj1027](https://osu.ppy.sh/users/7547506) and [Abstract-](https://osu.ppy.sh/users/3097304) winning the previous year, some analysts had doubts that they could claim the winning title this year. Thankfully or not, that wasn't the case here at all. With 11 maps lost in this entire tournament, South Korea showed that even with not the strongest players this game could possibly meet there are plenty of players that can still impress in this mode.

Reviewer 2: ``` Suggestion
It is no secret that players from South Korea are so good that they could swap with each other and win the entire tournament no matter what they were faced up against. With [qwhj1027](https://osu.ppy.sh/users/7547506) and [Abstract-](https://osu.ppy.sh/users/3097304) not being in the South Korean team like the previous year, some analysts had doubts that they could claim the winning title this year. Thankfully or not, that wasn't the case here at all. With 11 maps lost in this entire tournament, South Korea showed that even without the strongest players this game could produce, there are plenty of players ready to take their shot at impressing the community and the crowd.
```

> United States entered this tournament with almost the same roster with swapping [Kisetsunotori](https://osu.ppy.sh/users/3692648) to [Elux](https://osu.ppy.sh/users/12004983) being the only roster change. Although they've fallen to the losers bracket after playing a match against South Korea, United States showed their determination to achieve something worthy. In the end, they've claimed themselfes as grand finalists for the second time in this world cup.

Reviewer 2: ``` Suggestion
The United States entered this tournament with almost the same roster with swapping [Kisetsunotori](https://osu.ppy.sh/users/3692648) to [Elux](https://osu.ppy.sh/users/12004983) being the only roster change. Although they've fallen to the losers bracket after playing a match against South Korea, the United States showed their determination to achieve something worthy. In the end, they've claimed themselves as grand finalists for the second time in this world cup.
```

> With the recent change of swapping from Group Stages to Qualifiers format, most of the players and viewers didn't expect to see China on a very top despite failing their previous run placing 8th-7th place. Their expectations worth it in the end as they instantly gained their glory reaching winners finals bracket without losing a single match until meeting with South Korea and untimately to United States. Even with a long year absense, China showed that even being absent for an entire year they can still make a good fight with top teams.

Reviewer 2: ``` Suggestion
With the recent change of swapping from Group Stages to Qualifiers format, most of the players and viewers didn't expect to see China on the top despite failing their previous run placing 7th—8th place. Their expectations were worth it in the end as they instantly gained their glory reaching the winner's finals bracket without losing a single match until clashing with South Korea and, ultimately, the United States. Even with a long year of absence, China showed that even being absent for an entire year they can still make a good fight with top teams.
```

> There's still plenty of tournaments running though even after the conclusion of CWC! The digit tournaments, [4 Digit Catch World Cup 2022](https://osu.ppy.sh/wiki/en/Tournaments/4CWC/2022) and [3 Digit Catch World Cup 2022](https://osu.ppy.sh/community/forums/topics/1580097 ), are up and running and a second iteration of [Catch French Dual Cup 2](https://osu.ppy.sh/community/forums/topics/1605746) currently in the registered phase. There's nothing to talk about these tournaments yet but maybe with the new osu!monthly release we can fully recap what was going on with conclusions and stuff!

Reviewer 2: ``` Suggestion
There are still plenty of tournaments running, even after the conclusion of CWC! The digit tournaments, [4 Digit Catch World Cup 2022](https://osu.ppy.sh/wiki/en/Tournaments/4CWC/2022) and [3 Digit Catch World Cup 2022](https://osu.ppy.sh/community/forums/topics/1580097 ), are up and running and the second iteration of [Catch French Dual Cup 2](https://osu.ppy.sh/community/forums/topics/1605746) is currently in the registering phase. There's nothing to talk about these tournaments just quite yet, but maybe with the new osu!monthly release we can fully recap what infamous rivalries transpire along with the conclusions, and maybe something new!
```

Author: a big one, nicely done

### ![](/wiki/shared/mode/mania.png) osu!mania

#### In August, the osu!mania top players will brawl in the osu!mania World Cup 2022!

> As it is starting the **6th of August**, we can't actually talk about it more, but we are excited to say that the next osu!monthly is going to have a **big article** about it, and maybe an **interview of the winner**!

Reviewer 2: ``` Suggestion
As it is starting the on **6th of August**, we can't actually talk about it more, but we are excited to say that the next osu!monthly is going to have a **big article** about it, and maybe an **interview of the winner**!
```

> Please stay tuned about the next article, and we're moving on the next section!

Reviewer 2: Not really sure this needs to be here, not sure what the purpose is of telling the reader the next article is coming when it is pretty visible that there is more coming.

Author: Thought it was too short, so I wrote this
I'm deleting it

<iframe width="100%" height="315" src="https://youtube.com/embed/HMQKYkffUjk" frameborder="0" allowfullscreen></iframe>

### ![](/wiki/shared/mode/taiko.png) osu!taiko
#### Whew! If you thought the pick for osu! was crazy, wait 'till you watch [shinchikuhome’s](https://osu.ppy.sh/users/3174184) DT play on [Apollo](https://osu.ppy.sh/beatmapsets/1739877#taiko/3558252)! 
This taiko play is set at  an average of 411 BPM, yes, you read that right! **411 BPM!**. This is truly an insane play for any speed enthusiasts and absolute taiko enjoyers. Make sure to show some love and enjoy the absolute speed demon plays shinichikuhome is known for!

<iframe width="100%" height="315" src="<link>https://youtu.be/embed/hKwIKC-DvcU" frameborder="0" allowfullscreen></iframe>

### ![](/wiki/shared/mode/mania.png) osu!mania

#### The mania k-pop community is going to be absolutely ecstatic at this next showcase. 

> With [Onta_Bekasi](https://osu.ppy.sh/users/8677684) not just clearing, but FCing the deceptively difficult yet soothing myriad of LNs and intricate patterns that [My Man](https://osu.ppy.sh/beatmapsets/514681#mania/1093212) so elegantly puts on display, it is of true bewilderment that [Onta_Bekasi](https://osu.ppy.sh/users/8677684) only suffers a 00.02 accuracy drop across a 3 minute map and over 5000 notes. Truly a wonderful display of skill, enjoyment of the game, and a play that is more than deserving of being showcased.

Reviewer 2: ``` Suggestion
With [Onta_Bekasi](https://osu.ppy.sh/users/8677684) not just clearing, but FCing the deceptively difficult yet soothing myriad of LNs and intricate patterns that [My Man](https://osu.ppy.sh/beatmapsets/514681#mania/1093212) so elegantly puts on display, it is of incredible skill, and no small effort, that [Onta_Bekasi](https://osu.ppy.sh/users/8677684) only suffers a 00.02 accuracy drop across a 3-minute map and over 5000 notes. Truly a wonderful display of what osu!mania can offer, enjoyment of the game, and is a play that is more than deserving of being showcased.
```

<iframe width="100%" height="315" src="<mania 7k>https://youtu.be/embed/1RuyJ3y_ot4" frameborder="0" allowfullscreen></iframe>

#### Wow. Just completely shocked at this performance by [[GS]Myuka](https://osu.ppy.sh/users/10072733) on [Camellia’s - MI11I0N PP](https://osu.ppy.sh/beatmapsets/1388481#mania/2867552). 

> If you thought *My Man* had LNs, wait until you watch just over 7 minutes of almost non-stop LN action along with being bridled by intense single-note action and BPM changes throughout the map. 

Reviewer 2: ``` Suggestion
If you thought *My Man* had LNs, wait until you see a map of over 7 minutes of non-stop LN gameplay, and with not just single-note rainbows and glissandos, but an interconnected web of the two. It is an impeccable display of skill, accuracy, speed, and just what osu!monthly is looking for in a showcase!
```

---
layout: post
title: osu!monthly #1
date: 2022-07-28
author: Soetch, Blushing, wowcake, 0x84f
---

![](/wiki/shared/news/2022-08-01-osumonthly-1/banner-july.png)

> # Introducing osu!monthly

Reviewer: ```suggestion
## Introducing osu!monthly
```
Level 1 headings don't seem to be used.

Author: Real, didn't see it.

It has been a long time since the last osu!weekly was released and luckily, we have something exciting to show you: osu!monthly! The new community news post format.

6 years passed in which the community didn't have a regular post. Today, we're coming in with ***osu!monthly***, a bigger format.

> ![](/wiki/shared/news/2022-08-01-osumonthly-1/banner-july.png)

Reviewer: This image is now there two times. Not sure if the position in the article makes a difference.

Author: Fixed.

## Word of the editors :

> It's been years since **osu!weekly's** release has been paused, 6 years since the community didn't have a regular post. Today, we're coming in with **__osu!monthly__**, a bigger format.

Reviewer: ```suggestion
```
My suggestion was to shorten this to the sentence above (`6 years passed in which...`) and have everything under `Introducing osu!monthly` (`word of the editors` needs to be removed too in this case)

Author: Doing this rn, sorry for not noticing.

Author: Actually, I'm keeping the 2 last paragraphs of it, see the last commit.

  
  [➥ Find the article here!](#if-you-ask-us-what-is-the-big-event-of-july-itd-be-the-roundtable)

- For the month's top score, we got [[Karcher]](https://osu.ppy.sh/users/11443437) achieving an insane Full Combo on [*Yomi yori Kikoyu, Koukoku no Tou to Honoo no Shoujo*](https://osu.ppy.sh/beatmapsets/461744#fruits/1031991)!
  
  [➥ Find the article here!](#as-if-the-impossible-could-not-be-done-only-once-but-twice)

### ![](/wiki/shared/mode/taiko.png) osu!taiko 
- [The Expert Global Taiko Showdown 2022](https://osu.ppy.sh/wiki/en/Tournaments/GTS/EGTS_2022) is hitting on the stage! This is the way Taiko players can overtake [The Roundtable](#if-you-ask-us-what-is-the-big-event-of-july-itd-be-the-roundtable) in terms of epic shows!
  
  [➥ Find the article here!](#as-if-we-could-allow-for-osu-to-take-all-the-glory-in-its-showcasing-of-top-players-and-their-performances-taiko-takes-it-upon-themself-to-rival-even-that-of-the-roundtable-with-their-very-own-annual-entry-of-expert-global-taiko-showdown-2022)

### ![](/wiki/shared/mode/catch.png) osu!catch

> - The [Catch World Cup 2022](https://osu.ppy.sh/wiki/en/Tournaments/CWC/2022) has ended on pretty amazing performances ! Is South Korea dominating two modes at the same time?

Reviewer 2: ``` Suggestion
- The [Catch World Cup 2022](https://osu.ppy.sh/wiki/en/Tournaments/CWC/2022) has ended on some pretty amazing performances! Is South Korea dominating two modes at the same time?
```

<iframe width="100%" height="315" src="https://www.youtube.com/embed/byyHOAopo1A" frameborder="0" allowfullscreen></iframe>

---

## New Features on osu!(lazer)!

The new client has never so **close** from releasing (there's still a lot of work on it!), and has **new mods**, more original than never! 
For the first time ever, lazer patch notes have been posted on the official osu! [YouTube channel](https://www.youtube.com/c/osugame), compiled by [pishifat](https://osu.ppy.sh/users/3178418) with further insight provided by the man himself, [peppy](https://osu.ppy.sh/users/2)!
Check the video out:

<iframe width="100%" height="315" src="https://www.youtube.com/embed/67P4LKlAh-0" frameborder="0" allowfullscreen></iframe>

If you are interested in all previous and future patch notes, you can find them [here](https://osu.ppy.sh/home/changelog).

One thing you might've missed is the new fancy Repel mod added to lazer, which is pretty chaotic but fun (finally a worthy opponent for [badeu](https://osu.ppy.sh/users/1473890)?)

<div align="center">
    <video width="100%" controls>

>         <source src="https://user-images.githubusercontent.com/47010459/172435116-6b09a48c-6fd6-4abb-9e5f-e96ab3565184.mp4" type="video/mp4" preload="none">

Reviewer 5: these need to be static hosted on assets.ppy.sh 100%, don't trust this link to stay active forever

will do this before release

Author: Oh, we were wondering about this, thanks for telling!

Reviewer 3: If it helps the video on line 173 was ripped from this [patch notes post](https://osu.ppy.sh/home/changelog/lazer/2021.514.0) but I couldn't find where the patch notes are held on Github so

    </video>
</div>

Another feature that's maybe not the newest, but some of you might have missed, or just haven't gotten around to trying yet is the **skin layout editor**! First implemented in May of 2021, this is still very much a work-in-progress feature, however currently you are able to edit the placement of all skin elements that appear during gameplay! Check out the showcase video from the original patch notes below:

<div align="center">
    <video width="100%" controls>
        <source src="https://user-images.githubusercontent.com/191335/118155608-5c5fce80-b453-11eb-88f4-e552b10ed5f1.mp4" type="video/mp4" preload="none">
    </video>
</div>

> ## Summary of this Article

Reviewer 5: on further thinking, we should do one of a few things:

1) remove the summary section entirely, since i don't expect it to get this much traffic this far inside the article as it stands
2) keep it, but condense it down significantly (mostly by changing the "article here" links to direct references to newsposts instead of sections inside the article)
3) as 2, but shift the section up to be the first actual content section under the table of contents so that it is encountered first

Author: Guess we'll remove it.

<iframe width="100%" height="315" src="<mania 4k>https://youtu.be/embed/nc_ejGtZKUM" frameborder="0" allowfullscreen></iframe>

### ![](/wiki/shared/mode/catch.png) osu!catch

#### Somebody scream! Even with being the number one osu!catch player in the world, a play such as [YesMyDarkness’](https://osu.ppy.sh/users/4158549/fruits) is no small feat. 

Coming in with a HD HR 4xmiss on [Kurokotei’s Galaxy Collapse](https://osu.ppy.sh/beatmapsets/521900#fruits/1108507) one of the most well known artists and songs in osu! surely does do them some justice. Riddled with intense and **dash**ingly hard jumps it is a play to contend with even the hardest of maps, let alone with hidden and hardrock! 

<iframe width="100%" height="315" src="https://www.youtube.com/embed/byyHOAopo1A" frameborder="0" allowfullscreen></iframe>

---
## New Features on osu!lazer!

The new client has never been that **close** from releasing (there's still a lot of work on it!), and has **new mods**, more original than never! 

> 

Reviewer 3: ```suggestion
For the first time ever, lazer patch notes have been posted on the official osu! [YouTube channel](https://www.youtube.com/c/osugame), compiled by [pishifat](https://osu.ppy.sh/users/3178418)and further insight provided by the man himself, [peppy](https://osu.ppy.sh/users/2)!
Check the video out:

<div align="center">
    <video width="100%" controls>
        <source src="https://www.youtube.com/watch?v=67P4LKlAh-0" type="video/mp4" preload="none">
    </video>
</div>
```

> Here are the [patch notes](https://osu.ppy.sh/home/changelog), and the showcase of **the latest mod**! 

Reviewer 3: ```suggestion
If you are interested in all previous and future patch notes, you can find them [here](https://osu.ppy.sh/home/changelog).
```

> To start with this, we want to showcase a pretty chaotic but fancy Repel mod (finally a worthy opponent for [badeu](https://osu.ppy.sh/users/1473890)?)! 

Reviewer 3: ```suggestion
One thing you might've missed is the new fancy Repel mod added to osu!, which is pretty chaotic but fun (finally a worthy opponent for [badeu](https://osu.ppy.sh/users/1473890)?)
```

<div align="center">
    <video width="100%" controls>
        <source src="https://user-images.githubusercontent.com/47010459/172435116-6b09a48c-6fd6-4abb-9e5f-e96ab3565184.mp4" type="video/mp4" preload="none">
    </video>
</div>

---

> Another new feature that some of you might not know of, or just haven't gotten around to trying yet is the **skin layout editor**! First implemented in May of 2021, this is very much a work-in-progress feature, however currently you are able to edit the placement of all skin elements that appear during gameplay!

Reviewer 3: ```suggestion
Another feature that's maybe not the newest, but some of you might have missed, or just haven't gotten around to trying yet is the **skin layout editor**! First implemented in May of 2021, this is still very much a work-in-progress feature, however currently you are able to edit the placement of all skin elements that appear during gameplay! Check out the showcase video from the original patch notes below:
```

#### EGTS 2022

Not content to allow osu! to vacuum up all the glory, the members of the osu!taiko community took it upon themselves to create a serious contender of their own with the annual running of the **[Expert Global Taiko Showdown 2022](/wiki/Tournaments/GTS/EGTS_2022)**!

Showcasing some of the best drum bashers the community has to offer, EGTS has a storied history of being one of the most competitively and visually impressive contests around, sporting enough dons and kats to even make the Taikonator blush.

> The GTS team's insanely talented designer ::{{ flag=PH }}:: [OsuMe65](https://osu.ppy.sh/users/852867), has also made a trailer for the tournament.Check it out and leave your thoughts down below!

Reviewer 6: ```suggestion
The GTS team's insanely talented designer ::{{ flag=PH }}:: [OsuMe65](https://osu.ppy.sh/users/852867), has also made a trailer for the tournament. Check it out and leave your thoughts down below!
```
